sim/warrior/colossus_smash.go

```diff
@@ -47,14 +47,13 @@ func (warrior *Warrior) RegisterColossusSmash() {
 			IgnoreHaste: true,
 		},
 
-		DamageMultiplier: 1.0,
+		DamageMultiplier: 1.5,
 		CritMultiplier:   warrior.DefaultMeleeCritMultiplier(),
 
 		BonusCoefficient: 1,
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-			baseDamage := 120.0 +
-				1.5*spell.Unit.MHNormalizedWeaponDamage(sim, spell.MeleeAttackPower())
+			baseDamage := 120.0 + spell.Unit.MHNormalizedWeaponDamage(sim, spell.MeleeAttackPower())
```

Comment:
I'm not sure for this as the tooltip damage should be already after the weapon multiplier - so 120/1.5 = 80 base. I tried opening the collosus smash in wago and I cant even see from where the base damage comes so im just confusing. Maybe how you have it is already correct :D